### PR TITLE
Wizard recipe: DynamO-v1.7.6

### PR DESCRIPTION
**Pull request by lucifer1004**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: DynamO
* Version: v1.7.6

@staticfloat please review and merge.

### CODE REVIEW

File: D/DynamO/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc")
+]
```
Comment (giordano) [inline D/DynamO/build_tarballs.jl:30]:
```suggestion
platforms = supported_platforms(; exclude=p -> libc(p) != "glibc")
```

File: D/DynamO/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+products = [
+    ExecutableProduct("dynarun", :dynarun),
+    ExecutableProduct("dynamod", :dynamod),
+    ExecutableProduct("dynapotential", :dynapotential),
+    ExecutableProduct("dynahist_rw", :dynahist_rw)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="boost_jll", uuid="28df3c45-c428-5900-9ff8-a3135698ca75"))
```
Comment (giordano) [inline D/DynamO/build_tarballs.jl:44]: #3024

Comment (lucifer1004) [reply D/DynamO/build_tarballs.jl]: Added compat.

File: D/DynamO/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+products = [
+    ExecutableProduct("dynarun", :dynarun),
+    ExecutableProduct("dynamod", :dynamod),
+    ExecutableProduct("dynapotential", :dynapotential),
+    ExecutableProduct("dynahist_rw", :dynahist_rw)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="boost_jll", uuid="28df3c45-c428-5900-9ff8-a3135698ca75", compat="=1.76.0"))
```
Comment (giordano) [inline D/DynamO/build_tarballs.jl:37]:
```suggestion
    Dependency(PackageSpec(name="boost_jll", uuid="28df3c45-c428-5900-9ff8-a3135698ca75"); compat="=1.76.0")
```

**Pull request merged**